grammars/xml.cson

```diff
@@ -343,3 +375,94 @@
         'name': 'punctuation.definition.comment.xml'
     'end': '--%?>'
     'name': 'comment.block.xml'
+  'embedded-css':
+    'patterns': [
+      {
+        'begin': """(?ix)
+          <style
+          (?=[>:\\s])
+          (?!
+            [^>]* \\s type
+            \\s* = \\s*
+            ['"]?
+            (?!\\btext/css\\b)
+            [^'"]+
+            ['"]?
+          )
+          [^>]*
+          >
+        """
+        'beginCaptures':
+          '0':
+            'patterns': [
+              'include': '#generic-xml'
+            ]
+        'contentName': 'source.css.embedded.xml'
+        'end': '(?ix:</style>)'
```

Comment:
If we match case-insensitively here then the beginCapture should as well.
Also, the `x` modifier is complete overkill.